grpc/src/client/load_balancing/child_manager.rs

```diff
@@ -219,6 +219,11 @@ impl<T: PartialEq + Hash + Eq + Send> LbPolicy for ChildManager<T> {
     fn work(&mut self, _channel_controller: &mut dyn ChannelController) {
         todo!();
     }
+
+    fn exit_idle(&mut self, channel_controller: &mut dyn ChannelController) {
+        todo!()
+        // TODO(cjzhao): add exit idle logic once everything has been merged in
```

Comment:
Note that the `todo!` macro accepts a string that is printed when it is encountered, which might be nicer.  Just `todo!("implement exit_idle")` seems fine.

Reply:
You can delete the comment here in that case.